content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -343,9 +343,7 @@ namespaces based on their labels.
 {{< feature-state for_k8s_version="1.22" state="stable" >}}
 
 The Kubernetes control plane sets an immutable label `kubernetes.io/metadata.name` on all
-namespaces, provided that the `NamespaceDefaultLabelName`
```

Comment:
removing this line is not correct as upper line is conditional

Reply:
I remove all the feature gate description and keep immutable label `kubernetes.io/metadata.name`, Is this OK?